integrated_channels/cornerstone/views.py

```diff
@@ -127,7 +127,11 @@ def get(self, request, *args, **kwargs):
 
         get_count = int(request.GET.get('count', exporter.MAX_PAYLOAD_COUNT))
         content_item_count = min(get_count, exporter.MAX_PAYLOAD_COUNT)
-        data = transmitter.transmit(*exporter.export_for_web_polling(max_payload_count=content_item_count))
+        created, updated, _ = transmitter.transmit(
```

Comment:
Cornerstone pulls content via an API call. In order to accomplish this in the past, it overrode a lot of the Transmittercode and added custom methods. I've tweaked the base implementation signature of transmit to return the lists of items it worked on (now that we save all content items). This allows us to call the normal transmit and work on the results rather than implement an alternative like we did in the past.

Reply:
In our discussions this week, it sounds like we'll want to include these deleted items but ensure they have an `IsActive` key set to `false` - an attempt at this is implemented with a field transformation, but its unclear if its working properly today.